lib/models/Compare_Crypto.rb

```diff
@@ -27,13 +26,20 @@ def display_mini_menu
         fiat_input = get_fiat_input
         puts CLEAR
         MainMenu.new.show_logo
-        set_crypto_1(crypto_inputs[0],fiat_input)
-        set_crypto_2(crypto_inputs[1],fiat_input)
+        crypto_inputs.each do |n|
+            set_crypto(n,fiat_input)
+        end
+        @crypto.each do |n|
+            calculate_historical_change(n[0][0],n[0][1],n[0],n[1])
+        end
+        time2 = Time.now
```

Comment:
Naming here, what is time2? The clearer the names the easier it is to revisit/for other devs to pick up later on